packages/account/src/Components/self-exclusion/self-exclusion.tsx

```diff
@@ -125,14 +193,14 @@ const SelfExclusion = ({
 
     React.useEffect(() => {
         if (typeof setIsOverlayShown === 'function') {
```

Comment:
I believe, if we point in types, that setIsOverlayShown is function, there is no necessity of this checking.